File: C#/백준/Bronze/10768. 특별한 날/특별한 날.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            int b = int.Parse(Console.ReadLine());
            if(a==1 || (a==2 && b<18)){
                Console.WriteLine("Before");
            }
            else if(a==2 && b==18){
                Console.WriteLine("Special");
            }
            else{
                Console.WriteLine("After");
            }
        }
    }
}

File: Java/백준/Bronze/10768. 특별한 날/특별한 날.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt(); int b= in.nextInt();
        if(a==1 || (a==2 && b<18)){
            System.out.println("Before");
        }
        else if(a==2 && b==18){
            System.out.println("Special");
        }
        else{
            System.out.println("After");
        }
    }
}